teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>How can you plot information for&nbsp;an additional molecule in the&nbsp;radar charts that we were created in this talktorial?</p><p>-&gt; How can you plot information for&nbsp;an additional molecule in the&nbsp;radar charts that we <u>have</u> created in this talktorial?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>